### ISSUE TITLE
prompt to detach domain when closing a mux client window

### ISSUE THREAD

**Issue by jsgf** [wez/wezterm#848]
**Is your feature request related to a problem? Please describe.**

When I hit X on a `wezterm connect` window connected to a mux server, it prompts
```
Really kill this window and all contained tabs and panes?
[Y]es [N]o
```

This gives me the impression that it will tear down server-side session state, which I never want to do -
I just want to close the local client window and leave the server state persistent and unaffected.

**Describe the solution you'd like**
The default should just close the local window. Alternatively it could add "detach" to the options,
making it clear that the server state is unaffected. But really I just want it to close the window unprompted
because tearing down server state is potentially very destructive and I wouldn't want it to be easy to
hit accidentally (ie, it could be functionality that's harder to reach than a UI element on the window
decoration).

**Describe alternatives you've considered**
Whenever I get that dialog I hit "[N]o" (or Esc) and kill the wezterm process from the command line.

**Comment by hgkamath** [comment]
I support the idea of attach/detach semantics for sessions/tabs. 
Additionally, I just made some sort of profound statement in #917.  
quote:  Perhaps window **SHOULD NOT BE** an axis of _wezterm-mux-server_ session-management, BUT it **SHOULD BE** a _client-side feature_ for tab organization.

**Comment by wez** [comment]
I'm consider this to be effectively "fixed" as of e9f1297
I consider the detach concept to be a separate thing from just not being horrible in this specific use case! :)
